Source/WTF/wtf/persistence/PersistentCoders.cpp

```diff
@@ -44,7 +44,7 @@ std::optional<AtomString> Coder<AtomString>::decode(Decoder& decoder)
     if (!string)
         return std::nullopt;
 
-    return {{ WTFMove(*string) }};
+    return { AtomString { WTFMove(*string) } };
```

Comment:
For short strings this is an inefficient idiom that first allocates a `String` and only then searches the `AtomString` table. I think it’s too good an optimization opportunity to pass by, so I’d kind of want a breadcrumb here.